modules/creating-kubefed-CRs.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * kubefed/installing-kubefed.adoc
+
+[id="creating-kubefed-CRs_{context}"]
+= Creating KubeFed custom resources
+
+You must create the `KubeFedWebHook` and `KubeFed` custom resources (CRs) for {KubeFedProductShortName} deployment.
+The {KubeFedProductName} operator will create the required custom resource definitions (CRDs) for these CRs automatically.
+
+.Prerequisites
+* You will need to have the `oc` CLI tool installed.
+
+.Procedure
+
+. Create a `KubeFedWebHook` resource to instantiate an admission webhook controller for KubeFed. You must create this CR with `scope: Cluster` to allow the webhook controller to serve API requests coming from all KubeFed control planes within the same cluster.
++
+----
+$ cat <<-EOF | oc apply -n <namespace> -f -
+ ---
+ apiVersion: operator.kubefed.io/v1alpha1
+ kind: KubeFedWebHook
+ metadata:
+   name: kubefed-webhook-resource
+ spec:
+   scope: Cluster
```

Comment:
I'm checking with Ivan if the kubefedwebhook.spec.scope is needed.

Reply:
@qinpingli @font any update on whether this is required?

Reply:
@abrennan89 the scope field is no needed.